### ISSUE TITLE
CASG: URGENT Peer Review Issue

### ISSUE THREAD

**Issue by mumezawa** [ecsion/eSeaGrant-Client-Bug-Tracker#276]
Hi, 

Jim Eckman, our director, and I have been testing the peer review function and there seems to be a bug.
His peer review comments are not showing up as a panel reviewer using the unique link, but shows up if you download the [peer comment](http://eseagrant2.ucsd.edu/index.php?module=RFP&view=Detail&mode=DownloadReport&record=13703&tab_label=RFP%20Reports) excel document. 

1) Here is the panel reviewer link: http://eseagrant2.ucsd.edu/operations.php?action=panel_reviewer&guid=NzhEMkU4QzYtNDY4QS00MDFDLTgzRjgtRDNBMTVEQTQ4MUMw&exuti=MTY5MjE=

2) The screenshot of what is available if you follow the link:
![screen shot 2016-07-05 at 12 51 31 pm](https://cloud.githubusercontent.com/assets/17860279/16597924/4e0f6134-42b0-11e6-805f-424c9b256f07.png)

3) What I see if you download the peer comments (see the last reviewer - Jim Eckman): 
![screen shot 2016-07-05 at 1 00 27 pm](https://cloud.githubusercontent.com/assets/17860279/16597961/78800a54-42b0-11e6-886e-2de1523c003d.png)

Please call me if this doesn't make sense (858-534-1160). 
The bottom line is that the not all peer reviews are showing up for panel reviewers. 
Thanks so much. 

Best regards,

Miho

**Comment by mumezawa** [comment]
User error... I figured it out.
Thanks!
